.github/workflows/test.yml

```diff
@@ -0,0 +1,17 @@
+name: Run tests
+on: [push]
+jobs:
+  run_tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v4
+      with:
+        python-version: '3.11'
+        cache: pip   # caching pip dependencies
+    - name: Install dependencies
```

Comment:
We should be running the build step here i.e. as per https://github.com/communitiesuk/funding-service-design-workflows/blob/e734a2b45471fb73038efe59c8cc74345181f638/.github/workflows/deploy.yml#L102-L106

Reply:
I have included the linked build step.